cpp/src/toppra/toppra.hpp

```diff
@@ -17,6 +17,7 @@
 #define TOPPRA_LOG_DEBUG(X) ((void)0)
 #endif
 
+#define TOPPRA_NEARLY_ZERO 1e-8
```

Comment:
Is this used somewhere ?

Reply:
I use this here https://github.com/hungpham2511/toppra/pull/129/files#diff-19af0bded9ec5a7749236d4bcce703caR18